numba/tests/test_array_reductions.py

```diff
@@ -148,6 +154,18 @@ def test_argmax_basic(self):
         npr, nbr = run_comparative(array_argmax, arr)
         self.assertPreciseEqual(npr, nbr)
 
+    def test_median_odd(self):
+        arr = np.arange(101)
+        np.random.shuffle(arr)
+        npr, nbr = run_comparative(array_argmax, arr)
```

Comment:
It seems I was really lousy with this PR, sorry will be more careful from
next time.

On 22 June 2015 at 19:48, Antoine Pitrou <notifications@github.com> wrote:

> In numba/tests/test_array_reductions.py
> <https://github.com/numba/numba/pull/1250#discussion_r32938067>:
>
> > @@ -148,6 +154,18 @@ def test_argmax_basic(self):
> >          npr, nbr = run_comparative(array_argmax, arr)
> >          self.assertPreciseEqual(npr, nbr)
> >
> > +    def test_median_odd(self):
> > +        arr = np.arange(101)
> > +        np.random.shuffle(arr)
> > +        npr, nbr = run_comparative(array_argmax, arr)
>
> Apparently you are not testing the right function :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/numba/numba/pull/1250/files#r32938067>.
>



-- 
Harsh